skimage/morphology/grey.py

```diff
@@ -123,17 +238,20 @@ def dilation(image, selem=None, out=None, shift_x=False, shift_y=False):
            [0, 0, 0, 0, 0]], dtype=uint8)
 
     """
-
-    if image is out:
-        raise NotImplementedError("In-place dilation not supported!")
-
-    image = img_as_ubyte(image)
-    selem = img_as_ubyte(selem)
-    return cmorph._dilate(image, selem, out=out,
-                          shift_x=shift_x, shift_y=shift_y)
+    selem = np.array(selem)
+    selem = _shift_selem(selem, shift_x, shift_y)
+    # invert the structuring element to patch the same thing happening inside
```

Comment:
Nope, patch. =P ie they invert the input structuring element (for some reason), so we invert ours before inputting it into their function.